python/sglang/srt/layers/quantization/int8_kernel.py

```diff
@@ -0,0 +1,52 @@
+import torch
```

Comment:
May we add `Adapted from https://github.com/InternLM/lmdeploy/blob/086481ed84b59bee3b8e4274e5fc69620040c048/lmdeploy/pytorch/kernels/cuda/w8a8_triton_kernels.py#L282`